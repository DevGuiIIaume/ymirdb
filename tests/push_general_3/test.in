set a 1
set b 2
push b a a
pop b
forward a
forward b
backward a
backward b
pop b
forward a
forward b
backward a
backward b
bye
