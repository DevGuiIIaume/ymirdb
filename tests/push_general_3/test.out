> ok

> ok

> ok

> a

> nil

> a

> b

> nil

> a

> nil

> nil

> nil

> nil

> bye
